vignettes/introduction.Rmd

```diff
@@ -0,0 +1,126 @@
+  %\VignetteIndexEntry{Using maicplus}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+<style type="text/css">
+
+body{ /* Normal  */
+      font-size: 14px;
+  }
+td {  /* Table  */
+  font-size: 10px;
+}
+h1.title {
+  font-size: 38px;
+}
+h1 { /* Header 1 */
+  font-size: 28px;
+  }
+h2 { /* Header 2 */
+    font-size: 22px;
+}
+h3 { /* Header 3 */
+  font-size: 18px;
+}
+code.r{ /* Code block */
+    font-size: 12px;
+}
+pre { /* Code block - determines code spacing between lines */
+    font-size: 14px;
+}
+</style>
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.dim = c(6, 4),
+  warning = FALSE
+)
+```
+
+
+# Introduction
+
+This package describes the steps required to perform a matching-adjusted indirect comparison (MAIC) analysis using the `maicplus` package in R where the endpoint of interest is either time-to-event (e.g. overall survival) or binary
+(e.g. objective tumor response).
+
+The methods described in this document are based on those originally described by Signorovitch et al. 2010 and described in the National Institute for Health and Care Excellence (NICE) Decision Support Unit (DSU) Technical Support
+Document (TSD) 18. [@signorovitch2010; @phillippo2016a]
+
+When there is no common comparator treatment to link a clinical trial of a new intervention to clinical trials of other comparator treatments, it is referred to as an unanchored MAIC. When there is a common comparator available to link, it is referred to as an anchored MAIC.
+
+The premise of MAIC methods is to adjust for between-trial differences in patient demographic or disease characteristics at baseline. When a common treatment comparator is unavailable, a MAIC assumes that differences between absolute outcomes that would be observed in each trial are entirely explained by imbalances in prognostic variables and effect modifiers. Under this assumption, every prognostic variable and every effect modifier that is imbalanced between the two studies must be available. This assumption is generally considered very difficult to meet. [@phillippo2016a] 
+
+There are several ways of identifying prognostic variables/effect modifiers to be used in the MAIC analyses. We can approach from a clinical perspective, seeking advice from clinical experts or looking into published paper or previous regulatory submissions. We can also approach it from a data perspective, exploring regression models or subgroup analysis to identify interactions between baseline characteristics and treatment effect.
+
+# Statistical theory behind MAIC
+
+We will briefly go over the statistical theory behind MAIC. For more detailed information, refer to Signorovitch et al. 2010. [@signorovitch2010]
+
+Let us define $t_i$ to be the treatment patient $i$ received. We assume $t_i=0$ if the patient received intervention (IPD) and $t_i=1$ if the patient received comparator treatment. The causal effect of treatment $T=0$ vs $T=1$ on the
```

Comment:
The matching is accomplished by re-weighting patients in the study with the IPD by their odds, or likelihood, of having been enrolled in the study with the AgD. We usually refer to probability with the term “likelihood” as we seek for the variable that would maximize the probability of observing the outcome. The approach is very similar to propensity score weighting with the difference that IPD is not available for one study, so the usual maximum likelihood approach cannot be used to estimate the parameters of the propensity score model. Instead, a method of moments must be used (the method of moments is a statistical method for the estimation of population parameters). After the matching is complete and weights have been added to the IPD, it is possible to estimate the weighted outcomes and compare the results across.

The mapping approach can be described as follows: assuming that each trial has one arm, each patient can be characterized by the following random triple (X, T, Y), where X represents the baseline characteristics (e.g., age, and weight), T represents the treatment of interest (e.g., T = 0 for the IPD study and T = 1 for the study with AgD), and Y is the outcome of interest (e.g., OS).

Each patient is characterized by a random triple (x_i, t_i, y_i) with i=1 to N but only when IPD is available, i.e., when t_i = 0. In case where t_i = 1, only the mean baseline characteristics \bar{x}_{(t_i=1)}=\sum_{(t_i=1)}x_i and mean outcome \bar{y}_{(t_i=1)}=\sum_{(t_i=1)}y_i are observed.

Given the observed data, the causal effect of treatment T = 0 versus T = 1 on the mean of Y can be estimated as follows: